code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/ContentCardSchemaData.java

```diff
@@ -69,6 +72,10 @@ public Object getContent() {
         return content;
     }
 
+    public String getContentJsonString() {
```

Comment:
Can we not expose this as a public API? Why will customers need to use the string content?